src/latexify/transformers/function_expander_test.py

```diff
@@ -0,0 +1,68 @@
+"""Tests for latexify.transformers.function_expander."""
+
+from __future__ import annotations
+
+import ast
+
+from latexify import parser, test_utils
+from latexify.test_stubs import hypot
```

Comment:
If we need a definition of hypot for some reason (e.g., to pass linters), defining it directly in this file would be better.